paket_stellar/__init__.py

```diff
@@ -11,10 +11,11 @@
 import util.logger
 
 LOGGER = util.logger.logging.getLogger('pkt.paket')
+DEBUG = bool(os.environ.get('PAKET_DEBUG'))
 BUL_TOKEN_CODE = 'BUL'
 ISSUER = os.environ['PAKET_USER_ISSUER']
 ISSUER_SEED = os.environ.get('PAKET_SEED_ISSUER')
-HORIZON = os.environ['PAKET_HORIZON_SERVER']
+HORIZON_SERVER = 'https://horizon-testnet.stellar.org' if DEBUG else 'https://horizon.stellar.org'
```

Comment:
I think I was a bit misleading here, but my intention was to use these defaults only if PAKET_HORIZON_SERVER is not set.